docs/.c4/views/functional-layers/layers.views.c4

```diff
@@ -0,0 +1,99 @@
+views "uds-core-functional-layers" {
+  // Base Layer - Istio, UDS Operator, Policy Engine
+  view base-layer of cloud.cluster {
+    title "UDS Core Base Layer"
+
+    include
+      cloud,
+      cluster,
+      uds-core,
+      istio-system.*,
+      istio-tenant-gateway.*,
+      istio-admin-gateway.*,
+      pepr-system.*,
+      zarf.*,
+      kubernetes-control-plane.*
+  }
+
+  // Identity and Authorization Layer - Keycloak, AuthService
+  view identity-authorization-layer of cloud.cluster {
+    title "UDS Core Identity & Authorization Layer"
+
+    include
+      cloud,
+      cluster,
+      admin,
+      enduser,
+      uds-core,
+      uds-core.keycloak.*,
+      uds-core.authservice.*,
+      uds-core.istio-tenant-gateway.*,
+      uds-core.istio-admin-gateway.*,
+      cloud.cluster.external-services.ext-keycloak-db
+  }
+
+  // Metrics Server Layer (if applicable)
+  view metrics-server-layer of cloud.cluster {
+    title "UDS Core Metrics Server Layer"
+
+    include
+      cloud,
+      cluster,
+      uds-core,
+      kubernetes-control-plane.*
+  }
+
+  // Runtime Security Layer - Falco
+  view runtime-security-layer of cloud.cluster {
+    title "UDS Core Runtime Security Layer"
+
+    include
+      cloud,
+      cluster,
+      uds-core,
+      uds-core.falco.*,
+      uds-core.loki.*,
+      cloud.cluster.external-services.ext-loki-storage,
```

Comment:
Maybe a philosophy question on how we construct these/what to include - I think this currently is somewhat misleading as it shows Loki as part of the runtime-security layer. It is a _soft_ dependency so I can see why it might be included but I think it muddies the clarity on what is actually part of the layer itself? I wonder if grouping or different coloring might help?

Same would apply to some of the other layers too.